frontend/javascripts/viewer/controller/td_controller.tsx

```diff
@@ -77,6 +77,7 @@ function getTDViewMouseControlsSkeleton(planeView: PlaneView): Record<string, an
             event.ctrlKey || event.metaKey,
             OrthoViews.TDView,
             isTouch,
+            false,
```

Comment:
this sets `allowNodeCreation` to false in the 3d viewport case (the user can never create new nodes there). previously, this caused a wrong branch to be taken inside the left click handler which is why node selection did not work.